### ISSUE TITLE
coherence times

### ISSUE THREAD

**Issue by davidwhogg** [davidwhogg/HoneyComb#10]
Do the solar oscillations only have coherence times of days?  If so, we can treat each Kepler month independently and average the periodograms! (ish)

The giants have very long coherence times, but I hear that the solar-type stars not so much.  This could be useful, computationally.

**Comment by RuthAngus** [comment]
Yes, the Solar-type stars' modes only last a few days. Definitely much less than a Kepler quarter!

**Comment by davidwhogg** [comment]
As noted in [some other issue] I want to see this by seeing a zoom-in to one of the main modes in a G-dwarf
